ext/reflection/php_reflection.c

```diff
@@ -31,6 +31,9 @@
 
 #include "zend.h"
 #include "zend_API.h"
+#include "zend_language_scanner.h"
+#include "zend_language_scanner_defs.h"
```

Comment:
This file does not exist.

Reply:
it's weird... The same header file is included in the tokenizer extension and in Nikic's php-ast: https://github.com/nikic/php-ast/blob/bc4ccdc6279b0b862eb820ae1ac0f3265f6876d7/ast.c#L12 Do you have any clue what I did wrong here?

Reply:
These files are generated by running re2c on zend_language_scanner.l. Looks like a build order issue on Linux.

Reply:
I managed to solve the build issue by creating a `Makefile.frag` file with a similar content to the one in the tokenizer extension.

Reply:
Which functions do you need from this header? I would prefer moving them, or at least their declarations into zend_compile rather than dealing with these dependency issues here.